src/Libraries/Microsoft.Extensions.Http.Diagnostics/Logging/Internal/Log.cs

```diff
@@ -46,17 +46,114 @@ public static void OutgoingRequestError(ILogger logger, LogRecord record, Except
         OutgoingRequest(logger, LogLevel.Error, 2, nameof(OutgoingRequestError), record, exception);
     }
 
-    [LoggerMessage(LogLevel.Error, RequestReadErrorMessage)]
-    public static partial void RequestReadError(ILogger logger, Exception exception, HttpMethod httpMethod, string? httpHost, string? httpPath);
+    public static void RequestReadError(ILogger logger, Exception exception, HttpMethod method, string? host, string? path)
```

Comment:
New names of HttpClientLogging dimensions contain dots, e.g. `http.request.method`, but LoggerMessage currently don't support such parameters, because the name of a parameter must match the name of an argument of the logging method. Therefore, in order to create a log message with dimensions containing dots, we have to create the message manually.

The code below is a copy for code generated by `LoggerMessage`, I only made changes to the parameter names.

Reply:
Let's file a feature request to introduce a [TagName] attribute that the code generator will take the final name from.

Reply:
Create an issue https://github.com/dotnet/extensions/issues/4576.